DDG4/edm4hep/Geant4Output2EDM4hep.cpp

```diff
@@ -0,0 +1,589 @@
+//==========================================================================
+//  AIDA Detector description implementation 
+//--------------------------------------------------------------------------
+// Copyright (C) Organisation europeenne pour la Recherche nucleaire (CERN)
+// All rights reserved.
+//
+// For the licensing terms see $DD4hepINSTALL/LICENSE.
+// For the list of contributors see $DD4hepINSTALL/doc/CREDITS.
+//
+// Author     : F.Gaede, DESY
+//
+//==========================================================================
+
+#ifndef DD4HEP_DDG4_GEANT4OUTPUT2EDM4hep_H
+#define DD4HEP_DDG4_GEANT4OUTPUT2EDM4hep_H
+
+//  Framework include files
+#include "DD4hep/Detector.h"
+#include "DD4hep/VolumeManager.h"
+#include "DDG4/Geant4HitCollection.h"
+#include "DDG4/Geant4OutputAction.h"
+#include "DDG4/Geant4SensDetAction.h"
+#include "DDG4/EventParameters.h"
+
+// Geant4 headers
+#include "G4Threading.hh"
+#include "G4AutoLock.hh"
+#include "G4Version.hh"
+#include "G4SystemOfUnits.hh"
```

Comment:
```suggestion
#include "CLHEP/Units/SystemOfUnits.h"
```
Sorry I was unclear, I meant CLHEP. `G4SystemOfUnits` should never be used in conjunction with DD4hep.

Reply:
Why is that ? We are dealing with Geant4 here explicitly and not CLHEP - of course it is the same in the end, as
`G4SystemOfUnits.hh` includes the CLHEP header:

```c++
#include <CLHEP/Units/SystemOfUnits.h>
using CLHEP::millimeter;
using CLHEP::millimeter2;
...
```
so I find it more logical to document that we are using Geant4 units here ....

Reply:
.... ah, it is the namespace issue  - forget my last comment ;-)